src/Data/List/Utility.elm

```diff
@@ -0,0 +1,16 @@
+module Data.List.Utility exposing (splitFilter)
+
+
+splitFilter : (a -> Bool) -> List a -> ( List a, List a )
```

Comment:
This function already exists in `elm/core`:  https://package.elm-lang.org/packages/elm/core/latest/List#partition, please reuse that.